perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -383,7 +383,10 @@ public Void call(ApiCaller ac, Deserializer params) throws PerunException {
 
 			Member sponsoredMember = ac.getMemberById(params.readInt("member"));
 			User sponsor = ac.getUserById(params.readInt("sponsor"));
-			LocalDate newValidity = params.readLocalDate("validityTo");
+			LocalDate newValidity = null;
+			if (params.contains("validityTo")) {
+				newValidity = params.readLocalDate("validityTo");
```

Comment:
This will still fail, if the param is present, but the value is null, because of the way we parse date from the input.

Reply:
You are right, I forget about that. It should be fixed now.